src/EnergyPlus/DataRuntimeLanguage.hh

```diff
@@ -389,7 +390,8 @@ namespace DataRuntimeLanguage {
 			Variable( 0 ),
 			Expression( 0 ),
 			TrendVariable( false ),
-			TrendVarPointer( 0 )
+			TrendVarPointer( 0 ),
+			initialized( false )
 		{}
```

Comment:
As I understand it, only Plain Old Data need to be initialized in the constructors there.  Strings (and vectors, and... ) are more complex and, oldly enough, they don't need to be initialized there;  non-POD initialize on their own it seems.